### PR TITLE
Jake python2 demo notebooks

### PR DESCRIPTION
**Pull request by jakemulf**

### CODE REVIEW

File: demo/web/src/main/notebooks/A. Real-time table ops/A3 Do time series and relational joins.md [hunk 1]
```diff
@@ -23,26 +23,26 @@ For an `aj`, the values in the right table are matched to the closest values in
 Let's join these tables using an `aj` to get a single table with all of our information.
 
 ```python
-joined_data_aj = daily_data_0.aj(daily_data_1, "Timestamp", "Number1 = Number, Character1 = Character, Boolean1 = Boolean")
+joined_data_aj = daily_data_0.aj(table=daily_data_1, on=["Timestamp"], joins=["Number1 = Number", "Character1 = Character", "Boolean1 = Boolean"])
 ```
 
 Deephaven supports another type of as of join, an `raj`. For a `raj`, the values in the right table are matched to the closest values in the left table without going under the left value. For example, if the right table contains a value `5` and the left table contains values `4` and `6`, the right table's `5` will be matched on the left table's `4`.
```
Comment (margaretkennedy) [inline demo/web/src/main/notebooks/A. Real-time table ops/A3 Do time series and relational joins.md:29]:
```suggestion
Deephaven supports another type of as-of join, a `raj`. For a `raj`, the values in the right table are matched to the closest values in the left table without going under the left value. For example, if the right table contains a value `5` and the left table contains values `4` and `6`, the right table's `5` will be matched on the left table's `4`.
```

**Review by margaretkennedy — APPROVED** [review]

File: demo/web/src/main/notebooks/00 The Deephaven IDE.md [hunk 2]
```diff
@@ -57,7 +62,7 @@ updating_table = timeTable('00:00:00.400').updateView("Row = i", "Some_Int = (in
 3. [Kafka Stream vs Append](03%20Kafka%20Stream%20vs%20Append.md)
 
 
-Go to [quick start](https://deephaven.io/core/docs/tutorials/quickstart/) to install Deephaven from our pre-built images.
+Go to our [Quick start](https://deephaven.io/core/docs/tutorials/quickstart/) to install Deephaven from our pre-built images.
```
Comment (jjbrosnan) [inline demo/web/src/main/notebooks/00 The Deephaven IDE.md:65]:
```suggestion
Go to our [Quick start](https://deephaven.io/core/docs/tutorials/quickstart/) guide to install Deephaven from our pre-built images.
```

**Review by hythloda — APPROVED** [review]

**Review by jjbrosnan — APPROVED** [review]

File: demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md [hunk 3]
```diff
@@ -5,19 +5,19 @@ In this notebook, we show how to decorate and filter our data.
...
 
 ```python
-from deephaven.DateTimeUtils import currentTime, expressionToNanos, minus
+from deephaven.time import now, to_nanos, minus
 
-time_interval = expressionToNanos("T1M")
-offset = expressionToNanos("10D")
-now = currentTime()
+time_interval = to_nanos("00:01:00")
+offset = to_nanos("240:00:00")
```
[the remark this answers is not in the document]
Comment (jakemulf) [reply demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md:11]: Yeah unless I missed it I didn't see an `expressionToNanos` equivalent in the new API so 🤷

You could do `to_period` and there's probably a way to convert that period to nanos but that's not apparent right now

File: demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md [hunk 4]
```diff
@@ -37,18 +37,18 @@ With time series data, binning methods like [lowerBin](https://deephaven.io/core
...
+    agg.sum_(["Number"])
+]
 
-nanos_bin = expressionToNanos("T1M")
+nanos_bin = to_nanos("00:01:00")
 
-daily_data_binned = daily_data.update("TimestampMinute = lowerBin(Timestamp, nanos_bin)")\
-    .dropColumns("Timestamp")\
-    .aggBy(agg_list, "TimestampMinute")
+daily_data_binned = daily_data.update(formulas=["TimestampMinute = lowerBin(Timestamp, nanos_bin)"])\
```
[the remark this answers is not in the document]
Comment (jakemulf) [reply demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md:49]: I don't have a strong opinion either way

File: demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md (hunk 3 above)
[the remark this answers is not in the document]
Comment (hythloda) [reply demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md:11]: yes, agreed

**Pull request merged**